github/orgs_members_test.go

```diff
@@ -353,3 +354,81 @@ func TestOrganizationsService_RemoveOrgMembership(t *testing.T) {
 		t.Errorf("Organizations.RemoveOrgMembership returned error: %v", err)
 	}
 }
+
+func TestOrganizationsService_ListPendingOrgInvitations(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/orgs/1/invitations", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testFormValues(t, r, values{"page": "1"})
+		testHeader(t, r, "Accept", mediaTypeOrgMembershipPreview)
+		fmt.Fprint(w, `[
```

Comment:
@gmlewis, @shurcooL, As I mentioned above, I felt all fields need to be tested for assignment to the struct and compare values. Will it be ok only to test id field as before?